gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -32,124 +32,115 @@
+                args: ["{request}", "{flowManager}.lifecycleManager", "{arguments}.0"]
             },
             loginUser: {
-                funcName: "gpii.request.flowManager.userLogonStateChange.loginUser",
-                args: ["{that}", "{arguments}.0"],
-                dynamic: true
+                funcName: "gpii.flowManager.userLogonStateChange.loginUser",
+                args: ["{that}", "{arguments}.0"]
             },
             errorResponse: {
-                funcName: "gpii.request.flowManager.userLogonStateChange.errorResponse",
+                funcName: "gpii.flowManager.userLogonStateChange.errorResponse",
                 args: ["{request}", "{arguments}.0", "{arguments}.1"]
             }
         }
     });
 
-    fluid.defaults("gpii.request.flowManager.userLogonStateChange.matchMakingStateChangeHandler", {
-        gradeNames: ["gpii.request.flowManager.userLogonStateChange.stateChangeHandler",
-                     "gpii.request.flowManager.nonPrivacyFilteredMatchMakingRequest"],
+    fluid.defaults("gpii.flowManager.userLogonStateChange.matchMakingStateChangeHandler", {
+        gradeNames: ["gpii.flowManager.userLogonStateChange.stateChangeHandler",
+                     "gpii.flowManager.nonPrivacyFilteredMatchMakingRequest"],
         listeners: {
             onUserToken: "{that}.getDeviceContext",
             onMatchDone: "{that}.startLifecycle"
         }
     });
 
-    fluid.defaults("kettle.requests.request.handler.userLogonStateChange", {
-        gradeNames: ["fluid.littleComponent", "gpii.request.flowManager.userLogonStateChange.matchMakingStateChangeHandler"],
+    fluid.defaults("gpii.flowManager.userLogonStateChange.handler", {
+        gradeNames: ["kettle.request.http", "gpii.flowManager.userLogonStateChange.matchMakingStateChangeHandler"],
         invokers: {
-            handle: {
-                funcName: "gpii.request.flowManager.userLogonStateChange.logonChangeHandler",
+            handleRequest: {
+                funcName: "gpii.flowManager.userLogonStateChange.handleRequest",
                 args: ["{request}.req.params.userToken", "{flowManager}.lifecycleManager", "{that}"]
             }
         }
     });
+    
+    /**
+     * /user/:userToken/logonChange handler
+     * * If no user is logged in, user <userToken> will be logged in
+     * * If <userToken> is logged in, <userToken> will be logged out
+     * * If another user is logged in, nothing happens
+     * If "reset" is used as the <mytoken> for any of the above URLs, any currently logged in user will be logged out
+     */
+    gpii.flowManager.userLogonStateChange.handleRequest = function (userToken, lifecycleManager, that) {
+        // find currently logged in user:
+        var activeTokens = lifecycleManager.getActiveSessionTokens();
+        if (userToken === "reset") {
+            gpii.flowManager.userLogonStateChange.handleResetToken(activeTokens[0], that);
```

Comment:
The use of ```activeTokens[0]``` indicates only the first token in the ```activeTokens``` array will be reset rather than all active sessions? Does it mean the flow manager only supports one active session at a time?